### PR TITLE
liquidprompt 2.0.2

### PR DESCRIPTION
**Pull request by augmentedfourth**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by augmentedfourth** [comment]
The test failure seems to be related to the fact that Homebrew is attempting to execute the liquidprompt script as an executable, when it's meant to be sourced in the shell configuration. How can we resolve this?

**Comment by carlocab** [comment]
Perhaps with something like
```diff
diff --git a/Formula/liquidprompt.rb b/Formula/liquidprompt.rb
index 9c7be86208..fc9c703ad9 100644
--- a/Formula/liquidprompt.rb
+++ b/Formula/liquidprompt.rb
@@ -28,7 +28,11 @@ class Liquidprompt < Formula
 
   test do
     liquidprompt = "#{HOMEBREW_PREFIX}/share/liquidprompt"
-    output = shell_output("/bin/sh #{liquidprompt} 2>&1")
+    (testpath/"test.sh").write <<~SH
+      #!/bin/sh
+      source #{liquidprompt}
+    SH
+    output = shell_output("/bin/sh test.sh 2>&1")
     assert_match "add-zsh-hook: command not found", output
   end
 end
```

File: Formula/liquidprompt.rb
Comment (SMillerDev) [inline Formula/liquidprompt.rb:36]: It doesn't try to match anything so this is not a good test

**Comment by augmentedfourth** [comment]
2.0.1 was released before this got merged, so I've updated the commit and the PR.

File: Formula/liquidprompt.rb
Comment (carlocab) [inline Formula/liquidprompt.rb:32]: This is not a good test, as you're testing whether `/bin/sh` is reading commands correctly. So this is not a test of `liquidprompt` at all.

Comment (augmentedfourth) [reply Formula/liquidprompt.rb:32]: I agree, but the [original test](https://github.com/Homebrew/homebrew-core/commit/91a45347ef8efdc7ee2fa5247ceda6b04c634594) was dropped in by someone unfamiliar with `liquidprompt` and it was accepted. I just updated the assertion in accord with the newest version of the tool.

**Review by SMillerDev — APPROVED** [review]
Thanks @augmentedfourth  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by augmentedfourth** [comment]
@SMillerDev This is still tagged "do not merge." Can you remove that or do whatever else is necessary for this approved PR to merge?

Thanks!

**Pull request merged**
